CRVResponse/src/CrvCoincidenceClusterFinder_module.cc

```diff
@@ -217,6 +217,7 @@ namespace mu2e
             sumXY+=ht->_x*ht->_y;
           }
           //filled one full position&time cluster
+          if(layerSet.size()<2) continue;
```

Comment:
does 2 represent a coincidence count?  Could it ever change? ie could you imagine wanting to run with 2-bar coincidences, maybe in commissioning or OffSpill?  If so, please make this a configuration parameter.

Reply:
No, it represents the number of CRV layers. One needs to have at least 2 layers to form coincidences. We form coincidences based on 2, 3, or 4 layers.

Reply:
It's best to make this a variable declared constexpr.  When constexpr  it has the same run time speed as a literal value but the name of the variable lets you tell the story better.